### PR TITLE
[RW61X] Integrating code updates and CI workflow for RW61X platform

### PR DESCRIPTION
**Pull request by dinabenamar**
- Integrating code updates for RW61X platform and SDK 2.13.2 support
- Enabling RW61X platform in CI workflows to build all-clusters-app example

### CODE REVIEW

File: scripts/build/testdata/all_targets_linux_x64.txt [hunk 1]
```diff
@@ -14,6 +14,7 @@ linux-{x64,arm64}-{rpc-console,all-clusters,all-clusters-minimal,chip-tool,therm
 linux-x64-efr32-test-runner[-clang]
 imx-{chip-tool,lighting-app,thermostat,all-clusters-app,all-clusters-minimal-app,ota-provider-app}[-release]
 infineon-psoc6-{lock,light,all-clusters,all-clusters-minimal}[-ota][-updateimage]
+rw61x-all-clusters-app[-ota][-wifi][-thread][-factory-data][-matter-shell]
```
Comment (andy31415) [inline scripts/build/testdata/all_targets_linux_x64.txt:17]: I also see that laundry-washer-app and thermostat were changed. Should those also be compilable and available as targets?

Comment (andy31415) [reply scripts/build/testdata/all_targets_linux_x64.txt:17]: You seem to have modified the zap/matter files in those examples. Every example has a single zap file so I am unclear why something that only compiles for all-clusters required changes in zap files on laundry washer and thermostat - could you explain that part?

File: examples/laundry-washer-app/nxp/zap/BUILD.gn [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020 Project CHIP Authors
```
Comment (andy31415) [inline examples/laundry-washer-app/nxp/zap/BUILD.gn:1]: why were these added if there is no compile target for them?

File: scripts/build/testdata/all_targets_linux_x64.txt (hunk 1 above)
Comment (dinabenamar) [reply scripts/build/testdata/all_targets_linux_x64.txt:17]: @andy31415 sorry if that was unclear ! Currently, our all-clusters-app example is using the "examples/laundry-washer/nxp/zap/laundry-washer-app.zap" zap file by default, we still support switching back to "examples/all-clusters-app/all-clusters-common/all-clusters-app.zap" or "examples/thermostat/nxp/zap/thermostat_matter_wifi.zap" with compilation arguments. As of today, we don't have dedicated examples under laundry-washer and thermostat folders, we only have the all-clusters example which can be compiled as a laundry-washer (default) / thermostat / all-clusters device-types, only by changing the path to each corresponding zap file. We do plan to add dedicated examples for laundry-washer and thermostat apps under each folder, and then add each target in the workflows. Then, this current approach will be removed and we will keep only the default "examples/all-clusters-common/all-clusters-app.zap" file for the all-clusters example, and laundry-washer / thermostat examples will use each their dedicated zap file.

**Review by andy31415 — CHANGES_REQUESTED** [review]
Having all-clusters-app using the laundry-washer zap file is confusing. 

I expect example structure to be consistent - all-clusters-app should use an all-clusters specific zap file.

**Comment by dinabenamar** [comment]
> Having all-clusters-app using the laundry-washer zap file is confusing.
> 
> I expect example structure to be consistent - all-clusters-app should use an all-clusters specific zap file.

@andy31415 understood, thank you. We will work on enabling dedicated examples for laundry-washer & thermostat applications so that each can use their dedicated zap file, and keep the all-clusters application using the all-clusters zap file.
